rows.go

```diff
@@ -81,6 +82,15 @@ func (r *textRows) Next(dest []driver.Value) error {
 		return err
 	}
 
+	// skip row before WITH TOTALS,
+	// not but do not skip an empty line if it is part of the result
```

Comment:
please elaborate a little more. I don't quite understand the meaning

Reply:
If we read a line that contains nothing (condition `len(row) == 1 && row[0] == ""`), then two options are possible:

1. `SELECT query ''`
We need to keep this line as part of the output:
Let's check that we requested one item
`len(dest)! = 1` - the condition is not met, do not skip this data

2. This is a line break in queries like WITH TOTALS, WITH CUBE, WITH ROLLUP, where the aggregated part is separated from the main empty line:
It is important that such queries require groupings: `WITH TOTALS, ROLLUP or CUBE are not supported without aggregation`
In such queries, there will be more than one column, or it will not be empty. We skip this line in the results.